MCDataProducts/inc/KalSeedMC.hh

```diff
@@ -95,6 +104,7 @@ namespace mu2e {
     float _wireTau; // threshold cluster distance to the wire along the perpedicular particle path
     float _strawDOCA; // signed doca to straw
     float _strawPhi; // cylindrical phi from -pi to pi with 0 in Z direction
+    TrkStrawHitProvenance _provenance; // TODO default read value == Sim?
```

Comment:
A default constructor is now explicitly defined for the `TrkStrawHitMC` struct, which initializes the provenance to `External`.

Reply:
The default provenance has been updated to `Simulation`, so that existing simulation will be deserialized correctly.

Reply:
@kutschke The common `DigiProvenance` now inhabits a standalone src/inc pair.

Re default values: maybe I misunderstand your commentary, but the discussion here is about how to correctly initialize the provenance data member of `TrkStrawHitMC` when deserializing preexisting simulation in which the provenance field did not exist. Since the default value for the actual provenance enum will be `unknown`, as you asy, this needs to be explicitly overridden in the `TrkStrawHitMC` constructor to mark the preexisting simulation as having been produced as such.